providers/aws/services/account/account_security_questions_are_registered_in_the_aws_account/account_security_questions_are_registered_in_the_aws_account.py

```diff
@@ -1,13 +1,14 @@
-from colorama import Fore, Style
-
-from lib.check.models import Check
+from lib.check.models import Check, Check_Report
+from providers.aws.services.account.account_client import account_client
 
 # This check has no findings since it is manual
 
 
 class account_security_questions_are_registered_in_the_aws_account(Check):
     def execute(self):
-        print(
-            f"\t{Fore.YELLOW}INFO{Style.RESET_ALL} Manual check: Login to the AWS Console as root. Choose your account name on the top right of the window -> My Account -> Configure Security Challenge Questions."
-        )
-        return []
+        report = Check_Report(self.metadata)
+        report.region = account_client.region
+        report.resource_id = account_client.audited_account
+        report.status = "INFO"
+        report.status_extended = "Manual check: Login to the AWS Console as root. Choose your account name on the top right of the window -> My Account -> Configure Security Challenge Questions."
+        return [report]
```

Comment:
What happens right now with the info checks?

Reply:
It is treated as a normal finding.